src/timer/timer_game_calendar.h

```diff
@@ -75,11 +75,18 @@ class TimerGameCalendar {
 	struct TStorage {
 	};
 
-	static void SetDate(Date date, DateFract fract);
+	using Date      = int32;  ///< The type to store our dates in
+	using DateFract = uint16; ///< The fraction of a date we're in, i.e. the number of ticks since the last date changeover
+
+	using Year  = int32; ///< Type for the year, note: 0 based, i.e. starts at the year 0.
+	using Month = uint8; ///< Type for the month, note: 0 based, i.e. 0 = January, 11 = December.
+	using Day   = uint8; ///< Type for the day of the month, note: 1 based, first day of a month is 1.
+
+	static void SetDate(TimerGameCalendar::Date date, TimerGameCalendar::DateFract fract);
 
-	static Year year; ///< Current year, starting at 0.
+	static Year year;   ///< Current year, starting at 0.
 	static Month month; ///< Current month (0..11).
-	static Date date; ///< Current date in days (day counter).
+	static Date date;   ///< Current date in days (day counter).
```

Comment:
What are these random spaces?

Please don't do this .. it is such a nightmare. For example, why align 3 of the 4? That makes so little sense ...

Reply:
Year/Month/Day align with each other. 🙂

Reply:
But Yeah / Month / Date / DateFract doesn't. Not sure what DateFract did to you .. but yeah, please just don't. Either align everything in a single block, or don't align. This is just adding spaces for the fact of adding spaces :)